Chapter 7/services/todos/autohooks.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+const fp = require('fastify-plugin')
+const schemas = require('./schemas/loader')
+
+module.exports = fp(async function todoAutoHooks (fastify, opts) {
+  const todos = fastify.mongo.db.collection('todos')
+
+  fastify.register(schemas)
+
+  fastify.decorate('store', {
```

Comment:
I would move this decorator into the `mongo-data-source` plugin or into a dedicated file

Reply:
@fox1t I didn't mean to rename the decorator - I mean that the code in the `autohooks` file seems hard to find